config/development.json

```diff
@@ -126,6 +126,7 @@
 		"seller-experience": true,
 		"server-side-rendering": true,
 		"settings/security/monitor": true,
+		"stepper-woocommerce-poc": false,
```

Comment:
Should this be `true` since this is the config for development environments?

Reply:
Updated!